apps/patient/src/components/PharmacyCard.tsx

```diff
@@ -26,16 +29,71 @@ interface PharmacyCardProps {
     address: Address
     name: string
     distance: number
+    rating?: string
+    businessStatus: string
+    hours?: {
+      open?: boolean
+      is24Hr?: boolean
+      opens?: string
+      closes?: string
+    }
   }
   selected: boolean
   onSelect: Function
 }
 
+const Metadata1 = ({ businessStatus, rating, hours }) => {
```

Comment:
Can we give this a more descriptive name?

Reply:
Yeah, I was having trouble coming up with something better than Row1 lol. Any ideas?

Reply:
Ah, I'll do RatingHours

Reply:
udpated